iothub/service/src/Configurations/ConfigurationsClient.cs

```diff
@@ -42,12 +44,12 @@ internal ConfigurationsClient(string hostName, IotHubConnectionProperties creden
         }
 
         /// <summary>
-        /// Creates a new configuration for Azure IoT Edge in IoT hub
+        /// Creates a new <see cref="Configuration"/> for Azure IoT Edge in IoT hub.
```

Comment:
As the API definition includes a reference to this type, this reference here is unnecessary. I prefer the plain text that it was before. Same elsewhere.